playground.py

```diff
@@ -39,80 +41,65 @@ def p(info):
 file_id = None
 document_id = None
 
+
 def create_database():
     global database_id
 
     p("Running Create Database API")
     response = databases.create(
         database_id=ID.unique(),
-        name='Movies',
+        name="Movies",
     )
-    database_id = response['$id']
+    database_id = response["$id"]
     print(response)
 
+
 def create_collection():
     global collection_id
 
     p("Running Create Collection API")
     response = databases.create_collection(
         database_id,
         collection_id=ID.unique(),
-        name='Movies',
+        name="Movies",
         document_security=True,
         permissions=[
             Permission.read(Role.any()),
             Permission.create(Role.users()),
             Permission.update(Role.users()),
             Permission.delete(Role.users()),
-        ]
+        ],
     )
 
-    collection_id = response['$id']
+    collection_id = response["$id"]
     print(response)
 
     response = databases.create_string_attribute(
         database_id,
         collection_id,
-        key='name',
+        key="name",
         size=255,
         required=True,
     )
     print(response)
 
     response = databases.create_integer_attribute(
-        database_id,
-        collection_id,
-        key='release_year',
-        required=True,
-        min=0,
-        max=9999
+        database_id, collection_id, key="release_year", required=True, min=0, max=9999
     )
     print(response)
 
     response = databases.create_float_attribute(
-        database_id,
-        collection_id,
-        key='rating',
-        required=True,
-        min=0.0,
-        max=99.99
+        database_id, collection_id, key="rating", required=True, min=0.0, max=99.99
     )
     print(response)
 
     response = databases.create_boolean_attribute(
-        database_id,
-        collection_id,
-        key='kids',
-        required=True
+        database_id, collection_id, key="kids", required=True
     )
     print(response)
 
     response = databases.create_email_attribute(
-        database_id,
-        collection_id,
-        key='email',
-        required=False,
-        default=""
```

Comment:
This caused an error. Breaking?

Reply:
Should be fine since its a new major version